WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.kt

```diff
@@ -38,6 +38,8 @@ class HelpActivity : AppCompatActivity() {
         intent.extras?.get(WordPress.SITE) as SiteModel?
     }
 
+    private var mTicketsBeingShown: Boolean = false
```

Comment:
I already suggested a different solution that'd remove this line, but just to be through and in case that change doesn't happen, may I suggest dropping the `m` prefix from the property and renaming it to `ticketsBeingShown`. `m` prefix is a Java specific code style that's not used in Kotlin code.

Reply:
oops! thanks for pointing that out - and agreed your suggested solution is far clearer. Thanks for putting it together for me, going with it